doc/ref_model/chapters/chapter04.md

```diff
@@ -227,70 +226,12 @@ This section covers a list of implicit NFVI capabilities and measurements that d
 
 <p align="center"><b>Table 4-8:</b> Internal Measurement Capabilities of NFVI</p>
 
-<a name="4.1.4.5"></a>
-#### 4.1.4.5 Internal Security Capabilities
-
-Security content has been relocated to the Security chapter (RM Chapter 7).
-
-Table 4-9: Reserved
-
-<!--
-<a name="Table4-9"></a>
--->
-
 <a name="4.1.5"></a>
-### 4.1.5 Internal Infrastructure Performance Measurements
-<!--
-[COMMENT - Xavier Grall, Orange: section "3.4.2.3 Internal SLA metrics" is removed since it is redundant with network performance metrics]
-//
-[COMMENT - Xavier Grall, Orange: section "3.4.2.4 Internal scalability metrics" is removed since it is redundant with resource management metrics]
--->
-<a name="4.1.5.1"></a>
-#### 4.1.5.1 Internal Performance Measurements
-<!--
-[COMMENT - Xavier Grall, Orange: the mapping table is removed since those reference values will depend on architecture and implementation, and/or may be derived for different cases (eg w/ or w/o filtering rules for network throughput) ]
--->
-
-The following table of internal Performance Measurements shows PMs per NFVI node. Network test setups are aligned with ETSI GS NFV-TST 009 [2]. Specifically internal performance metrics use a baseline (Phy2Phy) dataplane test setup in a single host.
-
-<a name="Table4-10"></a>
-
-| Ref | NFVI Measurement | Unit | Definition/Notes |
-|--------------------|------------------------------------------------------|----------------|----------------------------------------------------------------------|
-| i.nfvi.pm.xxx | Place Holder | Units | Concise Description |
-
-<p align="center"><b>Table 4-10:</b> Internal Performance Measurements of NFVI</p>
-
-<a name="4.1.5.2"></a>
-#### 4.1.5.2 Internal Availability/Reliability Measurements
-
-<!-- Xavier Grall, Orange -->
-_**Editor Note:** the following table should be reviewed to only consider and probably detail the recovery-related PMs; indeed, availability and MTBF PMs do not seem consistent with expected testbed measurement duration]_
-
-<!--
-[COMMENT - Xavier Grall, Orange: the mapping table is removed since those reference values will depend on reference architecture and implementation]
--->
-
-<!-- MXS - 13/7/2019 To-do -->
-_**Editor Note:** This table needs to be reworked and clarified w/ clear explanations and assumptions stated._
-
-<a name="Table4-11"></a>
-
-| Ref | NFVI Measurement | Unit | Definition/Notes |
-|--------------------|------------------|---------|-------------------------------------------|
-| i.nfvi.pm.008 | Availability | % |  |
-| i.nfvi.pm.009 | MTBF single node | days | Mean Time between Failure for single node |
-| i.nfvi.pm.010 | MTBF AZ | days | Mean Time between Failure for an   AZ |
-| i.nfvi.pm.011 | Recovery time | seconds |  |
-
-<p align="center"><b>Table 4-11:</b> Internal availability/reliability metrics of NFVI</p>
-
-<a name="4.1.6"></a>
-### 4.1.6 VIM Capabilities.
+### 4.1.5 VIM Capabilities.
```

Comment:
Minor thing but could you remove the full stop from section title please?

Reply:
Done.